content/en/docs/contribute/new-content/new-features.md

```diff
@@ -98,7 +98,7 @@ deadlines.
 1. Open a pull request against the
 `dev-{{< skew nextMinorVersion >}}` branch in the `kubernetes/website` repository, with a small
 commit that you will amend later.
-2. Edit the pull request description to include links to `k/k` PR(s) and `k/enhancement` issue(s).
+2. Edit the pull request description to include links to `kubernetes/kubernetes` PR(s) and `kubernetes/enhancement` issue(s).
```

Comment:
Those repo names could be hyperlinks, if you like.

Reply:
_k/enhancement_ is actually https://github.com/kubernetes/enhancements (note plural)

Reply:
Updated! Thanks for catching that :D